build/loaders/gql-loader.js

```diff
@@ -10,7 +10,10 @@
 module.exports = function gqlLoader(content /*: string */) {
   // NOTE: For now, we are simply loading queries and schemas as strings.
   // However, we may wish to load a pre-parsed graphql AST, similar to how https://github.com/samsarahq/graphql-loader works.
-  const result = `module.exports = ${JSON.stringify(content.toString())};`;
+  const result = `
+  const gql = require('graphql-tag');
+  module.exports = gql(${JSON.stringify(content.toString())});
```

Comment:
Is the result of graphql-tag serializable? Could this be run at build-time?

Reply:
It might be - long term I do want to investigate that.